### ISSUE TITLE
[LDC] Dub doesn't work properly

### ISSUE THREAD

**Issue by CreatorVilius** [termux/termux-packages#4189]
**Problem description**
Dub doesn't work at all. No output, no input,  never stops until `ctrl+c`.

**Steps to reproduce**
- Install LDC.
- Create new directory.
- Type `dub init`.

**Expected behavior**
Outputs and inputs appear.

**Additional information**
No output.

**Comment by xeffyr** [comment]
Related to https://github.com/termux/termux-packages/issues/4185.

**Comment by CreatorVilius** [comment]
No, ldc2 command works as expected, but dub doesn't.

**Comment by xeffyr** [comment]
Most likely the root of both issues is one. Post output of `termux-info`.
I wasn't able to reproduce `dub` hang like you described on AArch64 / ARM devices.

**Comment by CreatorVilius** [comment]
```
Updatable packages:
All packages up to date
Subscribed repositories:
https://dl.bintray.com/grimler/science-packages-21 science/stable
https://dl.bintray.com/grimler/game-packages-21 games/stable
https://termux.net stable/main
https://termux.net stable/main
System information:
Linux localhost 3.4.0-4741109 #1 SMP PREEMPT Fri Jul 3 22:19:50 KST 2015 armv7l Android
Termux-packages arch:
arm
Android version:
5.0.2
Device manufacturer:
samsung
Device model:
SM-T530
```

**Comment by xeffyr** [comment]
On i686 android 5 `ldc2` crashes and `dub` hangs.

![Screenshot_1565897329](https://user-images.githubusercontent.com/25881154/63121025-35d5e080-bfac-11e9-9ac4-816df72d37e9.png)

Let's see if https://github.com/termux/termux-packages/pull/4191 will solve the issue.

**Comment by xeffyr** [comment]
Reopening since really different issue. https://github.com/termux/termux-packages/pull/4191 solves problem with `ldc2`, but not with `dub`.

**Comment by xeffyr** [comment]
Closing as fixed already in 1.18.0 (package is not available for `termux.net` though).
